src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -164,10 +180,7 @@ def get_organizations(
             else:
                 return list(Organization.objects.filter())
 
-        if user_id is not None:
-            qs = OrganizationMember.objects.filter(user_id=user_id)
-        else:
-            qs = OrganizationMember.objects.filter()
+        qs = OrganizationMember.objects.filter(user_id=user_id)
```

Comment:
Went ahead and made userId required, and assume that the user_id = None case is handled by some future separate method, in order to reduce risk of this method.